config/features/deprecated_flags.go

```diff
@@ -108,6 +108,11 @@ var (
 		Usage:  deprecatedUsage,
 		Hidden: true,
 	}
+	deprecatedP2PMetadata = &cli.StringFlag{
```

Comment:
great, thanks for the depreciation ( i was just going to write that we need to handle it)